<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="4" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>输入</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_delete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_eval">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>求值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_draw">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>作图</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit_main"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>value=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="output_value"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_x"/>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QTextBrowser" name="output_main"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>x=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QListWidget" name="listWidget_variables">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="actionbc"/>
    <addaction name="actiontc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actiongy"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionbc">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actiontc">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actiongy">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actiontc</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_main</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_input</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>972</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_x</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_eval</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>972</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
